pyrene/src/components/RadioGroup/RadioGroup.tsx

```diff
@@ -79,19 +79,19 @@ const RadioGroup: React.FC<RadioGroupProps> = ({
 
   return (
     <div
-      styleName="radioSelectionContainer"
+      className={styles.radioSelectionContainer}
       onBlur={onBlur}
       id={name}
     >
-      {title && <div styleName="radioGroupTitle">{title}</div>}
-      <div styleName={clsx('radioGroupContainer', { [`alignment-${alignment}`]: true })}>
+      {title && <div className={styles.radioGroupTitle}>{title}</div>}
+      <div className={clsx(styles.radioGroupContainer, { [styles[`alignment-${alignment}`]]: true })}>
```

Comment:
Please do the following when the `class` will be added anyway, means always `true`.

```
<div className={clsx(styles.radioGroupContainer, styles[`alignment-${alignment}`] )}>
```